cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -430,53 +461,58 @@ def test_provision_with_boot_volume(request, instance_args, provider, soft_asser
 
     image = inst_args.get('template_name')
 
-    with provider.mgmt.with_volume(1, imageRef=provider.mgmt.get_template_id(image)) as volume:
-        # Set up automate
-        method = modified_request_class.methods.instantiate(name="openstack_CustomizeRequest")
+    volume = provider.mgmt.create_volume(1, imageRef=provider.mgmt.get_template(image).uuid)
+    request.addfinalizer(lambda: provider.mgmt.delete_volume(volume))
+
+    # Set up automate
+    method = modified_request_class.methods.instantiate(name="openstack_CustomizeRequest")
+    with update(method):
+        method.script = dedent('''\
+            $evm.root["miq_provision"].set_option(
```

Comment:
If this is a hard-coded string, it'd be best to treat it like VOLUME_METHOD or ONE_FIELD and place it at the top of the file.

Reply:
This is questionable. This way you can easily see what is the  `method.script` set to, without having to scroll to the top of the file.

Reply:
In my opinion, it is better to follow the existing pattern of the codebase, where if you see something in all caps, you know that it is probably defined at the top of the file. This way, static variables are all in a single location for a test module.

Reply:
But they are gonna be used just once. There is no need to make define it and make it a constant. It would have a meaning if such constant would be used from many places, but this is not the case.

It is the same as lambda. Lambdas are unnamed functions that are passed to some method as callback targets. There is no need to name them just to be called, so it won't clutter the namespace and one doesn't need to scroll to see what the callback does.

Reply:
@jaryn I still think it is important to maintain consistency with the rest of the code base and test_module. You have two other methods strings `VOLUME_METHOD` and `ONE_FIELD`, why treat this method string differently? In addition, this approach is the one taken by the style guide https://www.python.org/dev/peps/pep-0008/#constants.

Reply:
I wouldn't call these symbols a constant and none of these does need to be on top of the module.

I didn't want to move things around much but OK. I will move the `ONE_FIELD` and `VOLUME_METHOD` to the placess where I think they do belong.